src/scalems/radical/runtime.py

```diff
@@ -76,6 +77,14 @@
     cache = functools.lru_cache(maxsize=None)
 
 
+def _parse_option(arg: str) -> tuple:
+    if not isinstance(arg, str):
+        raise InternalError('Bug: This function should only be called with a str.')
+    if arg.count('=') != 1:
+        raise argparse.ArgumentTypeError('Expected a key/value pair delimited by "=".')
+    return tuple(arg.split('='))
```

Comment:
This means that `PilotDescription()` would get a single string for any value.
Do we expect its type casting to be sufficient?
Is there a source of schema information that we could use before returning here to normalize the values?
Is this parsing a good idea, or should we use a set of manually coded argument handlers for the PilotDescription fields that we choose to expose on the command line?
Are there other possibilities?

Reply:
Note that `--pilot-option cores=1 --pilot-option gpus=0` has been added to the automated test and debug-level logging confirms it works as expected, so we could just accept this for now and refine later. But we can also reject now if the RCT team thinks this looks like an ugly technical can of worms, or if the other stakeholders don't like this interface idea.